core/video_options.hpp

```diff
@@ -49,7 +49,7 @@ struct VideoOptions : public Options
 			 "Create a new output file every time recording is paused and then resumed")
 			("segment", value<uint32_t>(&segment)->default_value(0),
 			 "Break the recording into files of approximately this many milliseconds")
-			("circular", value<bool>(&circular)->default_value(false)->implicit_value(true),
+			("circular", value<size_t>(&circular)->default_value(0)->implicit_value(1<<22),
 			 "Write output to a circular buffer which is saved on exit")
```

Comment:
Should we perhaps use megabyte units here?
Not too fussed either way.

Reply:
Yes, good suggestion, I think. I've done that now!